src/mame/rm/rmnimbus.h

```diff
@@ -103,9 +105,11 @@ class rmnimbus_state : public driver_device
 	required_device<z80sio_device> m_z80sio;
 	required_device<screen_device> m_screen;
 	required_ioport m_io_joystick0;
+	required_ioport m_io_joystick1;
 	required_ioport m_io_mouse_button;
 	required_ioport m_io_mousex;
 	required_ioport m_io_mousey;
+	ioport_port* m_io_selected_js;
```

Comment:
Using a pointer makes it harder to implement save state support.  Please use a `required_ioport_array<2>` for the joystick ports and keep an index for the selected port.

Reply:
Thank you, I will do that.